blockchain/src/history/mmr_store.rs

```diff
@@ -232,6 +241,72 @@ pub fn remove_block_from_store(
     }
 }
 
+/// Copies the mmr entries of the previous block number
+/// This is used to initialize the tree for the given block number, using the previous tree as a baseline
+pub fn init_mmr_from_block_number(
```

Comment:
What is the advantage of copying the contents over accessing the previous ones when needed?
Less complexity when accessing?

Reply:
See my comment above.